classification/metrics/compute_vessel_metrics.py

```diff
@@ -15,12 +15,12 @@
 
 Example:
 
+    python -m classification.metrics.compute_vessel_metrics \
+        --inference-table machine_learning_dev_ttl_120d.vessel_char_vid_features_v20190509  \
```

Comment:
@bitsofbits I would like to know a bit more about parameters, know which one is input, which one is output. For input, which points to where it is generated?